lib/config.yml

```diff
@@ -103,6 +103,9 @@ Layout/TrailingWhitespace:
 Layout/SpaceAroundMethodCallOperator:
   Enabled: true
 
+Lint/AmbiguousOperatorPrecedence:
+  Enabled: false
```

Comment:
This linter provides actual benefit since it can catch vague and subtle bugs with operator precedence.

Reply:
So you disagree with basically every programming language that implements BEDMAS (all of them)?

Ultimately I don't care so much anymore because I now have vscode auto correct this one for me. But seriously, this is some next level strictness.

Reply:
I believe that the team had generally settled on disabling style lints, but this is not a style lint, and as pointed out with my comment on one of our internal PRs, it can hide very legitimate and very subtle bugs.

I'm not saying computers don't understand operator precedence. The point of this linter rule is that humans often misunderstand operator precedence, and the intent of code and the actual implementation of said code can easily be different.